plugins/ibm_qradar/help.md

```diff
@@ -0,0 +1,162 @@
+# Description
+
+Orchestrate and automate ariel search and automate the offence management using the IBM Qradar
+
+# Key Features
+
+Identify key features of plugin.
+
+# Requirements
+
+* Example: Requires an API Key from the product
+* Example: API must be enabled on the Settings page in the product's user interface
+
+# Supported Product Versions
+
+_There are no supported product versions listed._
```

Comment:
Please add a list of versions of IBM QRadar that this plugin was tested with. According to [this documentation](https://www.ibm.com/docs/en/qsip) it appears that the most notable versions are 7.3.2, 7.3.3, and 7.4.